src/Calculator/SettingsPage.xaml

```diff
@@ -0,0 +1,164 @@
+        </Style>
+    </Page.Resources>
+
+    <ScrollViewer>
+        <Grid x:Name="PageGrid" AutomationProperties.LandmarkType="Main">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="1*"
+                               MinHeight="{StaticResource HamburgerHeight}"
+                               MaxHeight="52"/>
+                <RowDefinition Height="1*"/>
+            </Grid.RowDefinitions>
+
+            <Grid>
+                <local:TitleBar/>
+            </Grid>
+            
+            <Grid Height="{StaticResource HamburgerHeight}"
+                  HorizontalAlignment="Left"
+                  VerticalAlignment="Top"
+                  Grid.Row="1">
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="{StaticResource HamburgerHeightGridLength}"/>
+                    <ColumnDefinition Width="Auto"/>
+                </Grid.ColumnDefinitions>
+
+                <Button x:Name="BackButton"
+                            x:Uid="BackButton"
+                            Style="{StaticResource NavigationBackButtonNormalStyle}"
+                            Grid.Row="1"
+                            Click="BackButtonClick"/>
+
+                <TextBlock x:Uid="SettingsHeader"
+                                Grid.Column="1"
+                                Margin="8,-3,0,0"
+                                HorizontalAlignment="Left"
+                                VerticalAlignment="Center"
+                                Style="{StaticResource CategoryNameTextBlockStyle}"/>
+            </Grid>
+
+            <StackPanel Grid.Row="2" Margin="12,0,12,0">
+                <StackPanel x:Name="SettingsStack"
+                            AutomationProperties.LabeledBy="{Binding ElementName=SettingsAppTheme}"
+                            Orientation="Vertical">
+
+                    <TextBlock x:Name="SettingsAppTheme"
+                               x:Uid="SettingsAppTheme"
+                               Style="{StaticResource SubtitleTextBlockStyle}"
+                               AutomationProperties.HeadingLevel="Level1"
+                               TextTrimming="Clip"/>
+
+                    <RadioButton x:Name="SettingsLightTheme"
+                                 x:Uid="SettingsLightTheme"
+                                 Checked="ThemeChecked"
+                                 Tag="Light"/>
+
+                    <RadioButton x:Name="SettingsDarkTheme"
+                                 x:Uid="SettingsDarkTheme"
+                                 Checked="ThemeChecked"
+                                 Tag="Dark"/>
+
+                    <RadioButton x:Name="SettingsSystemTheme"
+                                 x:Uid="SettingsSystemTheme"
+                                 Checked="ThemeChecked"
+                                 Tag="System"/>
+
+
+                    <TextBlock x:Name="SettingsRestartApp"
```

Comment:
In light mode this is hard to read, might be reproed by switching to high contrast and back?
![image](https://user-images.githubusercontent.com/6334170/114793357-0c4ff800-9d3f-11eb-9f76-d952d79fb692.png)

Reply:
@joseartrivera Hmm, I'm not quite sure how to fix this, actually.

Reply:
Try making the foreground a ThemeResource instead of a StaticResource

Reply:
@joseartrivera Sorry for the late reply, I was busy lately. I'll try that.

Reply:
OK, this should be resolved now.